README.md

```diff
@@ -2,53 +2,66 @@
 Unofficial Ruby client for the Megaphone API
 
 ## Installation
-    gem 'megaphone_client', github:"scpr/megaphone_client"
+```bash
+gem 'megaphone_client', github:"scpr/megaphone_client"
+```
 
 ## Usage
+**Note:** Megaphone API props, such as `externalId`, is in camelCase instead of snake_case because Megaphone's API expects it when accessing their API. So when passing params or putting/posting a hash, use camelCase. When interfacing with the gem's API, use snake_case.
+
 ### Configuration
 Configure your app to connect to Megaphone, either in an initializer or your environment files:
 
 ```ruby
-  MegaphoneClient.setup do |config|
-      config.token = "{megaphone api token}"
-      config.network_id = "{megaphone network id}"
-      config.organization_id = "{megaphone organization id}"
-  end
+@megaphone = MegaphoneClient.new({
+  token: "{megaphone api token}",
+  network_id: "{megaphone network id}",
+  organization_id: "{megaphone organization id}"
+})
 ```
 
 ### Searching
 
+**Note:** the property `externalId` is written in camelCase because it's expected by the Megaphone API
+
 ```ruby
-Megaphone::Episode.search({ externalId: obj_key })
+@megaphone.episode.search({ externalId: obj_key })
 ```
 
 ### Updating
 
-Note: the properties in `body` are written in camelCase because it's expected by the Megaphone API
+**Note:** the properties in `body` are written in camelCase because it's expected by the Megaphone API
 
 ```ruby
-  Megaphone::Episode.update({
-    podcast_id: "{podcast id}",
-    episode_id: "{episode id}",
-    body: {
-      preCount: 1,
-      postCount: 2,
-      insertionPoints: ["10.1", "15.23", "18"]
-    }
-  })
+@megaphone.episode.update({
+  podcast_id: "{podcast id}",
+  episode_id: "{episode id}",
+  body: {
+    preCount: 1,
+    postCount: 2,
+    insertionPoints: ["10.1", "15.23", "18"]
+  }
+})
 ```
 
 ### Listing Podcasts
 
 ```ruby
-Megaphone::Podcast.list
+@megaphone.podcast.list
```

Comment:
same here, what about `@megaphone.podcasts.list` ?

Reply:
I completely agree on both the `podcast(s)` and `episode(s)` suggestion. I'll add that now